### PR TITLE
Fix css bugs

### PR DESCRIPTION
**Pull request by EMaksy**
# Description
Fixing two existing css bugs.

### Problem:  CTAs in Cluster detail view are not properly aligned
OLD unfixed:
[unfixed_cluster_details_view.webm](https://user-images.githubusercontent.com/54111255/219610400-b304324a-3ef5-4472-96fe-7328a01b2ea7.webm)

NEW fixed:
[fixed_cluster_details_view.webm](https://user-images.githubusercontent.com/54111255/219610366-7b09b358-426e-4a05-bbc7-eb833f07a3d7.webm)

### Problem: In the execution results page, the filter width  changes  if some values are selected

OLD:
old_filter

NEW:
fixed_filter_width

### CODE REVIEW

File: assets/js/components/ClusterDetails/ClusterDetails.jsx [hunk 1]
```diff
@@ -115,53 +115,60 @@ export function ClusterDetails() {
   return (
     <div>
       <BackButton url="/clusters">Back to Clusters</BackButton>
-      <div className="flex">
-        <PageHeader>
-          Pacemaker Cluster Details:{' '}
-          <span className="font-bold">{getClusterName(cluster)}</span>
-        </PageHeader>
+      <div className="flex flex-wrap  ">
+        <div className="flex w-1/2  h-auto overflow-hidden overflow-ellipsis  break-words ">
```
Comment (dottorblaster) [inline assets/js/components/ClusterDetails/ClusterDetails.jsx:119]: Would you remove these blank spaces?

**Review by dottorblaster — CHANGES_REQUESTED** [review]
Just some adjustments to do then if @jagabomb likes that I'm fine :grimacing:

**Review by nelsonkopliku — APPROVED** [review]
:rocket:

File: assets/js/components/ClusterDetails/ClusterDetails.jsx (hunk 1 above)
Comment (EMaksy) [reply assets/js/components/ClusterDetails/ClusterDetails.jsx:119]: Ah yeas thanks!

**Pull request merged**